site.yml

```diff
@@ -144,6 +144,13 @@
     - role: glance
       tags: ['openstack', 'glance', 'control']
 
+- name: nova code and config
+  hosts: controller:db:compute
+  vars_files:
+    - roles/rabbitmq/defaults/main.yml
+  roles:
+    - nova-common
```

Comment:
I don't understand this. Is there a reason to call this out specifically instead of having nova-common be an expressed dependency of the nova-something roles?